### DISCUSSION TITLE
Silence INFO messages

### DISCUSSION THREAD

**Discussion by MchlUh** [ourownstory/neural_prophet#1230] [Q&A - get help using NeuralProphet] [answered]
Hi, is there a common way to silence INFO in the fit method : 

INFO - (NP.df_utils._infer_frequency)
INFO - (NP.df_utils.return_df_in_original_format) 
INFO - (NP.config.set_auto_batch_epoch)

I am running a RandimozedSearchCV, and all the itermediate prints of neuralprophet are not helping track of the progress of the fine tuning process.

Thanks for any advice,
I tried this approaches, but unsuccessfully : 
https://stackoverflow.com/questions/29651505/disable-info-logging-messages-in-ipython-notebook

**Comment by noxan** [comment, marked as answer]
@MchlUh you can set the log level to `WARNING` or `ERROR` with the following snippet to hide `INFO` log entries.

```python
from neuralprophet import set_log_level
set_log_level("ERROR")
```
